### DISCUSSION TITLE
why are some sources empty ?

### DISCUSSION THREAD

**Discussion by alinaseer05** [lnreader/lnreader#470] [Q&A] [answered]
When I enter into some sources like reaper scans there isn't any novel

**Comment by rajarsheechatterjee** [comment, marked as answer]
The sources are empty as they could be broken. 

It could be due to many reasons like the source has updated their website's layout, added Cloudflare/anti scraping protection, etc.

**Reply by ghost** [reply]
I think the many sources has added anti scraping protection. 
Is there anything we can do to fix it?

**Reply by rajarsheechatterjee** [reply]
There's already a way to bypass Cloudflare by solving the Captcha in WebView. It would depend on the type of Anti Scraping Protection. Some can be bypassed in WebView whereas some might require a code change.
